### ISSUE TITLE
Link to sourcecode for modules of other Packages

### ISSUE THREAD

**Issue by Datseris** [JuliaDocs/Documenter.jl#1273]
Hi, in DynamicalSystems.jl 4 other Julia packages are loaded as modules, and the docs are being created. Here is how our make file looks like:

https://github.com/JuliaDynamics/DynamicalSystems.jl/blob/master/docs/make.jl

All functions that are expanded in `@docs` blocks come from modules that are re-exported by dynamical systems. We use `Reexport` to do this at the moment.

Unfortunately, the nice button "source" dissapears from the expanded docstrings. See e.g.:

https://juliadynamics.github.io/DynamicalSystems.jl/dev/chaos/lyapunovs/#ChaosTools.lyapunovs

Is it possible to restore this button to point at the source from another repo?

**Comment by goerz** [comment]
I've had the same problem with [QuantumControl](https://juliaquantumcontrol.github.io/QuantumControl.jl/dev/api/quantum_control/#Subpackages), and I finally figured out that Documenter shows the Source button if and only if the file with the source is in a git repository. This means I've had to install all the packages that I want to include docstrings for via `Pkg.develop` in the CI run that generates the documentation (which clones them into ~/.julia/dev` and thus allows Documenter to figure out the link to their original Github repo).

It would probably be good if this was explained in Documenter's documentation somewhere.
